### PR TITLE
Logs all the commands that have been used.

### PR DESCRIPTION
**Pull request by paullewis**
To support #362 :)

### CODE REVIEW

File: src/lib/drivers/cri.js [hunk 1]
```diff
@@ -140,6 +157,10 @@ class CriDriver extends Driver {
       throw new Error('connect() must be called before attempting to send a command.');
     }
 
+    if (this._commands.indexOf(command) === -1) {
+      this._commands.push(command);
```
Comment (samccone) [inline src/lib/drivers/cri.js]: if we use a set here we can drop the unique work

Comment (samccone) [inline src/lib/drivers/cri.js]: it will also prevent an unbounded growth of the commands arr

Comment (paullewis) [inline src/lib/drivers/cri.js]: ... It's kind of bound by the number of commands in the DevTools API :D But yeah ok.

**Comment by paulirish** [comment]
I dont know important this is to our typical users. I'd be happier to land this if we left it undocumented in the help

File: src/lib/drivers/cri.js (hunk 1 above)
Comment (samccone) [inline src/lib/drivers/cri.js]: :)

**Comment by paulirish** [comment]
I think kenneth is traveling, so all good. I'm going to close the PR for now. But its super easy to resurrect if it makes sense.

**Pull request closed without merging**